content-services/community/install/index.md

```diff
@@ -1,76 +1,69 @@
 ---
-title: Installing & Upgrading Content Services
+title: Install overview
 ---
 
-This section helps you to install Content Services, additional software, and modules. 
+This section helps you to install Alfresco Community Edition, additional software, and modules.
 
-## Introduction
+Community Edition consists of a number of modules that need to be deployed and configured together, to form the complete services platform. This provides customers with the flexibility to configure their deployment architecture in the way that is consistent with their IT strategy.
 
-Alfresco Content Services consists of a number of modules that need to be deployed and configured together, to form the complete services platform. This provides customers with the flexibility to configure their deployment architecture in the way that is consistent with their IT strategy.
+Alfresco deployment requires a number of infrastructure components to be in place, on which the core platform components are installed, and extended by several optional components which provide additional services.
 
-Alfresco deployment requires a number of infrastructure components to be in place, on which the core platform components are installed, and extended by a number of optional components which provide additional services. 
+The following diagram shows the key components of a typical Content Services installation:
 
-The following diagram illustrates the key components of a typical Content Services installation:
+![Typical Community Edition installation]({% link content-services/images/acs-typical-installation-community.png %}){:width="750px"}
 
-<img src="../../images/acs-typical-installation.png" width="750">
+Installation can be done manually, but Alfresco also provides a number of installation and deployment scripts for the most commonly used deployment tools to help customers accelerate the deployment process.
 
-Installation can be done manually, but Alfresco also provides a number of installation and deployment scripts for the most commonly used deployment tools, to help customers accelerate the deployment process. 
+## Install and deploy methods
 
-## Installation & Deployment Methods
-
-There are four distinct approaches to deploying Alfresco Content Services. Please familiarise yourself with Alfresco’s [Deployment Support Policy](https://docs.alfresco.com/support/latest/policies/containerization/), before selecting one of the options below.
+There are four distinct approaches to deploying Community Edition. Familiarize yourself with Alfresco's [Deployment Support Policy]({% link support/latest/policies/containerization.md %}), before selecting one of the options below.
```

Comment:
Why did we change that back? It need to point to Brian's newly revised deployment support page which superseded the containerisation one.

Reply:
It's not in _this branch_ as it'll break the links & the site build will fail (as it did in previous commits) so it'll put back in the next PR :-)